Userland/Demos/CatDog/CatDog.cpp

```diff
@@ -137,3 +137,9 @@ void CatDog::track_cursor_globally()
     set_global_cursor_tracking(true);
     GUI::WindowServerConnection::the().set_global_cursor_tracking(window_id, true);
 }
+
+void CatDog::context_menu_event(GUI::ContextMenuEvent& event)
+{
+    if (on_context_menu)
+        on_context_menu(event);
```

Comment:
This is called `on_context_menu_request` pretty much everywhere, please follow that pattern